compass/ocean/tests/global_ocean/mesh/arm10to60/dynamic_adjustment/__init__.py

```diff
@@ -0,0 +1,240 @@
+from compass.ocean.tests.global_ocean.dynamic_adjustment import \
+    DynamicAdjustment
+from compass.ocean.tests.global_ocean.forward import ForwardStep
+
+
+class ARM10to60DynamicAdjustment(DynamicAdjustment):
```

Comment:
This hope class was copied from the WC14 version and will need to be updated with an appropriate number of steps and appropriate parameter values.